src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(NUMBER));
+        return ball;
+    }
+}
```

Comment:
```suggestion
public class Lottery {
    private static final int NUMBER = 100;
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(NUMBER));
    }
}
```

that way you shouldn't get same generated values each time, and also you will create only 1 object per program (not each time you enter the method)